app/demo-geant-integration/selection.xml

```diff
@@ -0,0 +1,9 @@
+<lcgdict>
+    <class name="G4VHit" />
+    <class name="G4ThreeVector" />
+    <class name="demo_geant::HitData" />
+    <class name="demo_geant::HitRootEvent" />
+    <class name="demo_geant::SensitiveHit" />
+    <class name="std::vector<demo_geant::SensitiveHit*>" />
+    <class name="std::map<std::string, std::vector<G4VHit*> >" />
```

Comment:
Not strictly necessary but for completeness:
```suggestion
    <class name="std::vector<G4VHit*>" />
    <class name="std::map<std::string, std::vector<G4VHit*> >" />
```

Reply:
Added.